lib/ansible/module_utils/docker_common.py

```diff
@@ -457,6 +457,43 @@ def get_container(self, name=None):
 
         return result
 
+    def get_network(self, name=None):
+        '''
+        Lookup a network and return the inspection results.
+        '''
+        if name is None:
+            return None
+
+        result = None
+        try:
+            for network in self.networks():
+                self.log("testing network: %s" % (network['Name']))
+                if name == network['Name']:
+                    result = network
+                    break
+                if network['Id'].startswith(name):
+                    result = network
+                    break
+                if network['Id'] == name:
```

Comment:
That's not needed; if they are equal, `network['Id'].startswith(name)` was already `True`.